app/store/services/helpers/gps.js

```diff
@@ -16,42 +16,42 @@ const computeDistance = (
   return R * c / 1000;
 };
 
-const isInArea = (
-  position = { longitude: 0, latitude: 0 },
-  point = { longitude: 0, latitude: 0 },
+const isInRegion = (
+  currentLocation = { longitude: 0, latitude: 0 },
+  reference = { longitude: 0, latitude: 0 },
   Δλ = 0,
   Δφ = 0
 ) =>
```

Comment:
i am lost here. reference isn't region ?

Reply:
reference, here, is the reference position/location to be tested against current position/location. 
Combined with Δλ and Δφ, this should define an area.